### DISCUSSION TITLE
How to enable build for all in town?

### DISCUSSION THREAD

**Discussion by TheSirCororo** [TownyAdvanced/Towny#5682] [Q&A]
I use command /ta town <town> set perm build off but it not works. /t info says that build enabled for all, /plot info says build enabled only for residents

**Comment by LlmDl** [comment]
Use `/ta town TOWNNAME set perm off` and then `/ta town TOWNNAME set perm reset`

**Comment by TheSirCororo** [comment]
Don't works

**Comment by TheSirCororo** [comment]
All plots has another settings

**Comment by LlmDl** [comment]
If my first suggestion doesn't work tell the mayor to use `/t set perm off` `/t set perm reset`. If that doesn't work file a bug report.

**Comment by TheSirCororo** [comment]
Me need to ask all mayors of all towns do this? You're seriously?

**Reply by LlmDl** [reply]
Honestly, you've probably put Towny permission allowing everyone to build into your permission plugin *or* you've set the default town plot permissions in the config to allow this in the first place. What you're seeing is not normal behaviour and is due to mis-configuration.

**Reply by 1Pootis1** [reply]
good evening, my employee wrote above, I am in charge of the server, I provide the towny configuration, please see what is wrong there
https://pastebin.com/3p9x7jm6

**Reply by LlmDl** [reply]
File a [ticket.](https://github.com/TownyAdvanced/Towny/issues/new?assignees=&labels=bug&template=bug_report.yml)
